### ISSUE TITLE
Bug in Self Supervised Pretraining

### ISSUE THREAD

**Issue by EdvardGrodem** [Project-MONAI/tutorials#552]
During pretraining the validation loss is not correctly calculated. 
https://github.com/Project-MONAI/tutorials/blob/e1088488c1bcd9c53eaebcdc8b727fe98c2463b9/self_supervised_pretraining/ssl_script_train.py#L211

I belive it should be 
`total_val_loss /= val_step`

**Comment by yiheng-wang-nv** [comment]
Correct, thanks @EdvardGrodem
